projects/plugins/beta/docs/testing/testing-tips.md

```diff
@@ -8,7 +8,7 @@ To test out Jetpack, we recommend setting up a publicly accessible test site. Te
 
 **Jetpack Live Branches** is a [Tampermonkey](https://tampermonkey.net/) script that allows us to launch WordPress Sites with a Jetpack version coming from a changeset present on a Pull Request. This is achieved by installing the [Jetpack Beta Tester plugin](https://github.com/Automattic/jetpack-beta) on a fresh new WordPress site and switching to the branch matching the Pull Request.
 
-More details on how to use in the [Jetpack Live Branches README](https://github.com/Automattic/jetpack/blob/trunk/tools/jetpack-live-branches/README.md#readme).
+More details on how to use in the [Jetpack Live Branches README](https://github.com/Automattic/jetpack/blob/trunk/tools/jetpack-live-branches/README.md).
```

Comment:
Might want to keep the anchor here. It tries to skip past all GitHub's headers to get to the actual document.
```suggestion
More details on how to use in the [Jetpack Live Branches README](https://github.com/Automattic/jetpack/blob/trunk/tools/jetpack-live-branches/README.md#readme).
```